R/colour-manip.r

```diff
@@ -50,6 +50,9 @@ muted <- function(colour, l=30, c=70) col2hcl(colour, l = l, c = c)
 #' alpha("red", 0.1)
 #' alpha(colours(), 0.5)
 #' alpha("red", seq(0, 1, length.out = 10))
+#' alpha(c(`1` = "gold", `2` = "lightgray", `3` = "#cd7f32"), .5)
```

Comment:
I think you only need a single example here

Reply:
@hadley, I thought it would be helpful for the example to show (a) backticks, (b) quoted, and (c) unquoted. 

Does that change your mind?  If you still think the triplicate is tedious, I'll remove two.  Which line(s) are the preferred tidyverse style?  

I think I'd prefer quotes here, because sometimes these levels are assigned dynamically inside a function.

Reply:
I think `c("a" = "b")` is the best style. There's no need to elucidate every feature of `c()` here.